### PR TITLE
feat(create-atomic): replace variables for continious deployment

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/CDX-1098

## Proposed changes

There's 2 ways to deploy to netlify:

1. Link to a Github (& others) repo & enable continuous deployment
2. Do a manual deployement with `netlify deploy`

Basically the generated project did not work with 1)

## Breaking changes

None

## Testing

We don't test netlify deployment

### CODE REVIEW

File: packages/atomic/template/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@stencil/core": "2.17.1"
   },
   "devDependencies": {
+    "@rollup/plugin-replace": "4.0.0",
```
Comment (ThibodeauJF) [inline packages/atomic/template/package.json:16]: Ran into some install issues https://coveo.slack.com/archives/G01L9SYPNS0/p1659644197389519

File: packages/atomic/template/stencil.config.ts [hunk 2]
```diff
@@ -31,7 +32,13 @@ export const config: Config = {
   devServer: {
     port: parseInt(process.env.STENCIL_PORT) || 3333,
   },
-  plugins: [dotenvPlugin()],
+  plugins: [
+    replace({
+      'process.env.PLATFORM_URL': `'${process.env.PLATFORM_URL}'`,
+      'process.env.ORGANIZATION_ID': `'${process.env.ORGANIZATION_ID}'`,
+    }),
+    dotenvPlugin(),
```
Comment (ThibodeauJF) [inline packages/atomic/template/stencil.config.ts:40]: This is still useful for dev & manual deploys, which read content from the .env file.

File: packages/atomic/template/stencil.config.ts [hunk 3]
```diff
@@ -31,7 +32,13 @@ export const config: Config = {
   devServer: {
     port: parseInt(process.env.STENCIL_PORT) || 3333,
   },
-  plugins: [dotenvPlugin()],
+  plugins: [
+    replace({
+      'process.env.PLATFORM_URL': `'${process.env.PLATFORM_URL}'`,
+      'process.env.ORGANIZATION_ID': `'${process.env.ORGANIZATION_ID}'`,
```
Comment (ThibodeauJF) [inline packages/atomic/template/stencil.config.ts:38]: These 2 variables are needed for the Stencil app (the other variables are needed by the lambda function only)

**Review by jpmarceau — APPROVED** [review]
Thank you 🙏

File: packages/atomic/create-atomic/scripts/packageTemplate.json [hunk 4]
```diff
@@ -14,9 +14,9 @@
     "start": "node start-netlify.mjs",
     "build": "stencil build",
     "dev:live": "npm run site:link && node start-netlify.mjs --live",
-    "site:init": "netlify init",
-    "site:link": "netlify link",
```
Comment (ThibodeauJF) [inline packages/atomic/create-atomic/scripts/packageTemplate.json:18]: not really a use for this one

**Review by olamothe — APPROVED** [review]

**Pull request merged**